### ISSUE TITLE
Update Vento to 0.7.x

### ISSUE THREAD

**Issue by pluiedev** [lumeland/lume#478]
Currently (Lume v1.18.4) the Vento plugin uses Vento 0.6.x, which is missing a bunch of features (e.g. functions). Updating that should be trivial and would really make the lives of people who are actually trying out Vento (like myself) easier.

**Comment by oscarotero** [comment]
Yes, I'm planning to release a new version of Lume anytime soon, with all dependencies updated to the latest version.
In the meantime, you can use the import maps to change the version:

```json
{
  "imports": {
    "lume/": "https://deno.land/x/lume@v1.18.4/",
    "https://deno.land/x/vento@v0.6.0/": "https://deno.land/x/vento@v0.7.1/"
  }
}
```

**Comment by oscarotero** [comment]
@pluiedev you can use the latest development version of Lume with `deno task lume upgrade --dev`.

**Comment by oscarotero** [comment]
Lume 1.18.5 was released today, which includes Vento 0.7.1.
